### DISCUSSION TITLE
[Suggestion] Deprecate mingw32

### DISCUSSION THREAD

**Discussion by TheMadHau5** [msys2/MINGW-packages#8219] [Ideas]
As it seems that new subsystems (ucrt, clang and soon arm) are being added, all 64-bit, it seems logical to deprecate the mingw32 subsystem in favour of mingw64, as the former is barely used any longer. Once this is final, it can then be removed. This should also relieve some load off the package maintainers.

**Comment by Biswa96** [comment]
The time has not arrived yet 😌

**Comment by 1480c1** [comment]
Maybe better to switch to discussion format (ping @lazka)

I'm hesitant to say that we should on the account that even though it definitely would reduce maintenance burden on a lot of people, it may cause a lot of issues as there are still some people who use the 32-bit toolchain and packages because their downstream users still might use 32-bit binaries

**Comment by lazka** [comment]
We can just mark packages 64bit only in case upstream drops support or building no longer works for us.

**Comment by TheMadHau5** [comment]
Oh hey, I didn't know Github had discussions! Yeah, for now seems like that'll do. I wanted to bring this to notice because a lot of software is moving to only support amd64 and is abandoning x86.
